queue/amqp.go

```diff
@@ -304,6 +311,8 @@ func (q *AMQPQueue) RepublishBuried() error {
 	defer iter.Close()
 
 	retries := 0
+	notComplying := make([]*Job, 0, 3)
```

Comment:
3 for any specific reason?

Reply:
just `var something []Foo`, it won't make much difference in terms of memory usage and/or speed and you don't have to worry about which number to pick